js/models/actor.js

```diff
@@ -116,12 +116,18 @@ var Actor = Backbone.Model.extend({
     }
   },
 
-  removeTransientFeatures: function() {
-    var newFeatures = _.reject(this.get('features'), function(f) {
-      return (f != 'persistent');
-    });
-    this.save({
-      'features': newFeatures
+  // removeTransientFeatures: function() {
+  //   var newFeatures = _.reject(this.get('features'), function(f) {
+  //     return (f != 'persistent');
+  //   });
+  //   this.save({
+  //     'features': newFeatures
+  //   });
+  // },
+
+  resetFeatures: function() {
+    var newFeatures = ['available'];
+    this.save({'features': newFeatures
```

Comment:
The intent of the `<shift> + r`  `Reset Session` was to do the following.
1. Clear all conditions not marked `Persist last condition` see the 'o' command
  Visualized by underlined text.
2. If the the actor itself is not marked as `persistent`  see `<shift> + p`
  Remove the actor entirely. 

As far as the `available` feature or it's companion `reacted`, I really don't have a preference for setting those on reset. 

However, if the check for persistence isn't maintained the entire actor list would be removed entirely on two back to back resets.

 Lets say you have 8 primary members all marked and persistent members, which you should only have to do once. And then a battle ensues with 10 additional Kobolds. You would enter all 10 as non persistent. Once out of initiative, you can reset the entire list, maintaining primary members, and trimming out the rest with one command.